extensions/oidc/runtime/src/main/java/io/quarkus/oidc/SessionContent.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkus.oidc;
+
+/**
+ * Authorization Code Flow Session content
+ */
+public class SessionContent {
```

Comment:
Perhaps `SessionState` ?

Reply:
Hi Pedro @pedroigor So, as said below, the idea is that the session manager returns a `state` which encapsulates somehow the tokens, the default one returns them concatenated by default, but the DB one will return a reference to them etc.

Perhaps instead of `SessionContent` we should simply have `AuthorizationCodeTokens` ? `SessionManager` is translating `AuthorizationCodeTokens` into a session state.